Modules/Core/Common/include/itkConstNeighborhoodIterator.h

```diff
@@ -388,14 +388,7 @@ class ITK_TEMPLATE_EXPORT ConstNeighborhoodIterator
     return it.GetCenterPointer() == this->GetCenterPointer();
   }
 
-  /** Returns a boolean != comparison of the memory addresses of the center
-   * elements of two ConstNeighborhoodIterators of like pixel type and
-   * dimensionality.  The radii of the iterators are ignored. */
-  bool
-  operator!=(const Self & it) const
-  {
-    return it.GetCenterPointer() != this->GetCenterPointer();
-  }
+  ITK_DEFINE_INEQUAL_OPERATOR_MEMBER_FUNCTION(Self);
```

Comment:
Should we retain docstrings?

Reply:
Personally I would prefer to simply remove the (existing) documentation of `operator!=`, whenever it just matches the documentation of the corresponding `operator==`, without adding any extra information. As in this case:

https://github.com/InsightSoftwareConsortium/ITK/blob/v5.2.0/Modules/Core/Common/include/itkConstNeighborhoodIterator.h#L382-L384

Versus

https://github.com/InsightSoftwareConsortium/ITK/blob/v5.2.0/Modules/Core/Common/include/itkConstNeighborhoodIterator.h#L391-L393

I wouldn't mind adding a "standard" comment to the macro call, though, if you want to. For example:

     /** Defines operator!= simply by calling operator== */
     ITK_DEFINE_INEQUAL_OPERATOR_MEMBER_FUNCTION(Self);

What do you think?

Reply:
> adding a "standard" comment to the macro

This is probably better.